src/gui/attributetable/qgsattributetableview.h

```diff
@@ -80,6 +82,13 @@ class GUI_EXPORT QgsAttributeTableView : public QTableView
      */
     void setAttributeTableConfig( const QgsAttributeTableConfig &config );
 
+    /**
+     * Returns the selected features in the attribute table in sorted order.
```

Comment:
I'd just expand on this a bit - sorted order is a bit vague. E.g.

   Returns the selected features in the attribute table, in the order that they are shown in the table (i.e. respecting the current attribute table column sorting preferences).